.github/workflows/pitests.yml

```diff
@@ -0,0 +1,635 @@
+    steps:
+      - name: Setup local maven cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: checkstyle-maven-cache-${{ hashFiles('**/pom.xml') }}
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: pitest-javadoc
+        run: |
+          ./.ci/pitest.sh pitest-javadoc
+      - name: Stage results
+        if: failure()
+        run: |
+          mkdir staging && cp -r target/pit-reports/ staging
+      - name: Archive code coverage results
+        if: failure()
+        uses: actions/upload-artifact@v2
+        with:
+          name: pitest2-javadoc-coverage-report
+          path: staging
+          retention-days: 3
+
+  pitest-indentation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup local maven cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: checkstyle-maven-cache-${{ hashFiles('**/pom.xml') }}
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: pitest-indentation
+        run: |
+          ./.ci/pitest.sh pitest-indentation
+      - name: Stage results
+        if: failure()
+        run: |
+          mkdir staging && cp -r target/pit-reports/ staging
+      - name: Archive code coverage results
+        if: failure()
+        uses: actions/upload-artifact@v2
+        with:
+          name: pitest2-indentation-coverage-report
+          path: staging
+          retention-days: 3
+
+  pitest-xpath:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup local maven cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: checkstyle-maven-cache-${{ hashFiles('**/pom.xml') }}
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: pitest-xpath
+        run: |
+          ./.ci/pitest.sh pitest-xpath
+      - name: Stage results
+        if: failure()
+        run: |
+          mkdir staging && cp -r target/pit-reports/ staging
+      - name: Archive code coverage results
+        if: failure()
+        uses: actions/upload-artifact@v2
+        with:
+          name: pitest2-xpath-coverage-report
+          path: staging
+          retention-days: 3
+
+  pitest-packagenamesloader:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup local maven cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: checkstyle-maven-cache-${{ hashFiles('**/pom.xml') }}
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: pitest-packagenamesloader
+        run: |
+          ./.ci/pitest.sh pitest-packagenamesloader
+      - name: Stage results
+        if: failure()
+        run: |
+          mkdir staging && cp -r target/pit-reports/ staging
+      - name: Archive code coverage results
+        if: failure()
+        uses: actions/upload-artifact@v2
+        with:
+          name: pitest5-packagenamesloader-coverage-report
+          path: staging
```

Comment:
`retention-days: 3`

please add in all other places.

Reply:
Done